src/TKey.cpp

```diff
@@ -71,24 +72,33 @@ void TKey::setName(const QString& name)
     mpHost->getKeyUnit()->mLookupTable.insertMulti(name, this);
 }
 
-// This method is potentially flawed in that only the first child match is ever executed...!
-bool TKey::match(int key, int modifier)
+bool TKey::match(int key, int modifier, const bool isToMatchAll)
```

Comment:
Wouldn't `matchAll` be more much more readable?

Reply:
That is my *ism* kicking into play - you known, having all boolean variables begin with *is*!

**Is** it a problem for you? :wink:

Reply:
It just looks strange and I don't see why the simplest variable type is so special. I like code I can read naturally, not with Yoda-like grammar.